### PR TITLE
style: remove the extra space between end and loop

### PR DESCRIPTION
**Pull request by tristan-f-r**
noticed this! :heart:

### CODE REVIEW

**Comment by tristan-f-r** [comment]
hey can you remove the newline at the end of the&nbsp; file

**Comment by tristan-f-r** [comment]
@atalii

File: src/ahtml-node.adb [hunk 1]
```diff
@@ -141,4 +141,4 @@ package body AHTML.Node is
        Attrs => Attrs_Vec.Empty_Vector,
        Children => Index_Vec.Empty_Vector));
 
-end AHTML.Node;
```
Comment (atalii) [inline src/ahtml-node.adb:144]: Noticed you've gotten rid of the newline. Could you amend this and force push? Thanks!

**Review by atalii — CHANGES_REQUESTED** [review]

File: src/ahtml-node.adb (hunk 1 above)
Comment (tristan-f-r) [reply src/ahtml-node.adb:144]: hi! i use svn2git and force push doesn't work there

Comment (atalii) [reply src/ahtml-node.adb:144]: fascinating

**Pull request closed without merging**